package.json

```diff
@@ -11,9 +11,15 @@
     "format": "prettier . --write"
   },
   "dependencies": {
-    "@emotion/react": "^11.4.0",
-    "@emotion/styled": "^11.3.0",
+    "@emotion/react": "^11.7.1",
+    "@emotion/server": "^11.4.0",
+    "@emotion/styled": "^11.6.0",
     "@material-ui/core": "^5.0.0-beta.0",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/styles": "^4.11.4",
+    "@mui/icons-material": "^5.2.1",
+    "@mui/material": "^5.2.6",
+    "@mui/styles": "^5.2.3",
     "bootstrap": "^4.6.0",
```

Comment:
I think we can remove this dependency as well.

Reply:
I think we should keep this for now as other parts of the app are still using bootstrap.